File: static/mservice_template.py
import aiothings as aiot
import os

# os.environ['INPUT_MESSAGE_TOPIC']
# os.environ['OUTPUT_MESSAGE_TOPIC']
# event['sender']

def handler(event, context):
    # TODO implement
    aiot.consoleOutput('Console display:' + os.environ['MSERVICE_NAME']) # Output to console
    aiot.messagePublish({'data': event['data']}) # Must be a JSON, forward data as output message

File: static/mservice_template.js
const aiot = require('aiothings');

// process.env.INPUT_MESSAGE_TOPIC
// process.env.OUTPUT_MESSAGE_TOPIC
// event.sender

exports.handler = async (event, context) => {
    // Add your code here
    await aiot.consoleOutput('console display:' + process.env.MSERVICE_NAME); // display from console
	await aiot.messagePublish({data: 'payload'}); // must be a JSON
};